### ISSUE TITLE
Alpha Preview Build

### ISSUE THREAD

**Issue by Jeeppler** [unknown-horizons/godot-port#72]
Porting unknown-horizons to Godot is a pretty big task and it will take a long time. However, I would love to see very soon a highly experimental alpha-preview-build. Just a build that one can try. It does not have to do much. Even just displaying the menu and a map is fine, but having that build of the early early stage would give user a glimpse into the future of Unknown Horizons and potentially attract new developers.

**Comment by LinuxDonald** [comment]
I don't think that we will do this year any release. Maybe we will do an really small preview release next year. I don't want release something only to show how the main menu looks like. Devs and player want to see more. And that is what I want to show to others with an release.

**Comment by Jeeppler** [comment]
A small preview release next year would be good. Just showing the menu is an exaggeration. Surly, the release should be able to do a bit more. My point however is, do not wait a long time, instead release a version with a minimum set of features. Define what those features are, so that developers have a goal worth achieving.

**Comment by SeanKuehl** [comment]
Maybe make the goal for the preview just enough features to complete the first few objectives. Just a few buildings and/or units. What would this look like? Also, how long until you want the release?

**Comment by LinuxDonald** [comment]
I don't want set an date for an release. I will do it if we have something that we can show the players and interested developers.
Maybe some tasks could be:
Player controlled Ships
Players can build some buildings that do something. (like fisher goes fisching)
Maybe some land units (military)
